pkg/nodeconfig/mount.go

```diff
@@ -25,7 +25,7 @@ func MountPerNodeConfig(dcConfig *cassandra.DatacenterConfig) {
 }
 
 const (
-	defaultPerNodeConfigInitContainerImage = "mikefarah/yq:4"
+	defaultPerNodeConfigInitContainerImage = "k8ssandra/k8ssandra-client:v0.6.3"
```

Comment:
issue: So, I commented this on another ticket also, but no. The way to do this properly for future references (and removing everything from a CRD) would be the following one:

```go
"github.com/k8ssandra/cass-operator/pkg/images"

images.GetClientImage()
```

This configures correctly all the registryOverrides, namespace overrides and similar things. And takes care of updating this image. I know it's an older image in the currently used cass-operator, but we can create 1.23.3 with an updated one if you want (and Helm update is even easier).